AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -145,6 +148,15 @@ - (BOOL)isAppSecretRequired {
   return NO;
 }
 
+- (void)updateConfigurationWithAppSecret:(NSString *)appSecret transmissionTargetToken:(NSString *)token {
+  [super updateConfigurationWithAppSecret:appSecret transmissionTargetToken:token];
+
+  // Create the default target if not already created in start.
+  if (token && !self.defaultTransmissionTarget) {
```

Comment:
Why do you need this one? The callback will only be called when the SDK is started from library and then starting from application. `self.defaultTransmissionTarget` would always be `nil`.

Reply:
Oops. it is a target instance not target token. NM.